### PR TITLE
Add section headers to identify laws and other editorial changes in this context

### PR DESCRIPTION
**Pull request by SebastianZimmeck**
PR for #69.

### CODE REVIEW

**Comment by SebastianZimmeck** [comment]
Per @martinthomson's [suggestion](https://github.com/privacycg/gpc-spec/issues/69#issue-2174843185), I added section headers for the different laws and made a few other small editorial changes, none of which are intended to change any meaning. @j-br0, can you review if everything still stands as you intended?

File: index.html [hunk 1]
```diff
@@ -399,70 +399,66 @@ <h2>Legal Effects</h2>
...
+        extension or global device setting.” Under the CDPA, the GPC signal will be intended to 
+        communicate a request to opt out of both the sale of their personal information and the 
+        use of their personal information for targeted advertising.
+      </p>
+      <h3>Nevada Revised Statutes Chapter 603A (NRS 603A)</h3>
+      <p>
+        Under NRS 603A, a GPC signal will be intended to communicate a Do Not Sell My Personal
+        Information request [[?SB220]].
+      </p>
+      <h3>General Data Protection Regulation (GDPR)</h3>
```
Comment (jyasskin) [inline index.html:438]:
```suggestion
      <h3>European General Data Protection Regulation (GDPR)</h3>
```

I think? Or "EU" or similar?

**Review by jyasskin — APPROVED** [review]
I didn't review that the content of the sections is correct or matches the original content, but +1 to the change in structure.

**Comment by anderagakura** [comment]
I have a question also link to @AramZS [comment](https://github.com/privacycg/gpc-spec/pull/68#issuecomment-1984206085) in PR #68 : Is there no a risk of discrepancy about regional privacy regulations between the header from browser and a publisher with their CMP?

File: index.html (hunk 1 above)
Comment (SebastianZimmeck) [reply index.html]: Thank you, @jyasskin. I added "EU."

**Comment by SebastianZimmeck** [comment]
> I have a question also link to @AramZS [comment](https://github.com/privacycg/gpc-spec/pull/68#issuecomment-1984206085) in PR #68 : Is there no a risk of discrepancy about regional privacy regulations between the header from browser and a publisher with their CMP?

Most CMPs implement GPC and set other privacy signals accordingly. For example, if GPC is turned on (via the header or JS property), the IAB's GPP `usca` is set accordingly by the CMP. I have not done exhaustive studies on this point, but in general it should be the case. Any particular CMP should be able to answer that question for their platform.

**Comment by anderagakura** [comment]
@SebastianZimmeck I understand the fact that the CMP should handle the signal but : 

1. You mention GPP spec
2. What about TCF in UE which does not support GPC?

**Review by j-br0 — APPROVED** [review]

**Pull request merged**